### ISSUE TITLE
More UI setups.

### ISSUE THREAD

**Issue by Pranab009** [enricocid/Music-Player-GO#315]
1) **Describe the solution you'd like**
Please add more now playing themes.
- only ONE feature
- don't ask black theme or a different colors scheme as we have to follow guidelines for accessibility.
It is now playing theme.
2) **Describe alternatives you've considered*
Other players supported 1 more themes.

3) **Additional context**
At least add 1 or 2 themes for change only.
Every time you see same theme.
Please try to this features.
![Uploading 
![Screenshot_20210829-084917](https://user-images.githubusercontent.com/65340361/133187688-b60a185e-f205-4e90-8331-8e2b30693b34.png)

**Comment by enricocid** [comment]
I'll keep UI simple. :)
